pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -658,7 +658,7 @@ func (m *kubeGenericRuntimeManager) killContainer(pod *v1.Pod, containerID kubec
 			"containerName", containerName, "containerID", containerID.String(), "gracePeriod", gracePeriod)
 	}
 
-	klog.V(2).InfoS("Killing container with a grace period override", "pod", klog.KObj(pod), "podUID", pod.UID,
+	klog.V(2).InfoS("Killing container with a grace period", "pod", klog.KObj(pod), "podUID", pod.UID,
```

Comment:
Why have we removed the word "override" here?

Reply:
gracePeriodOverride logic is in  https://github.com/kubernetes/kubernetes/blob/9809038728c56e646e05430904fb7e01e89ce473/pkg/kubelet/kuberuntime/kuberuntime_container.go#L657, i think this line is not override.